content/en.json

```diff
@@ -69,12 +69,8 @@
       "subtitle": "We work for <em>the client of our client</em>. They use software to achieve <em>their</em> goals, our goal is to help them with that.",
       "how-title": "How we like to do this:",
       "how": [
-        "<em>Direct contact</em> with users",
-        "Prototyping, iterate quickly",
-        "Work user-centric",
-        "Close collaboration with designers",
-        "Work should deliver <em>value</em>",
-        "Work from \"why\""
```

Comment:
I'm not sure if I like these changes, it is not just "more brief" but it looses meaning. Specifically:

- "direct feedback" is not the same as "direct contact", feedback is great but too often it passes through many intermediaries, so the ultimate goal is "direct contact" and _one_ of the reasons is to get feedback (which I reckon we could leave out of the final text to be brief)
- "prototyping" is an explicitly mentioned example, I personally would prefer to keep it in there
- - "user-centric" is a [specific approach](https://en.wikipedia.org/wiki/User-centered_design) that we'd consider and which I'd like to keep in the list of "How", we could consider adding the link in there?
- "close collab with designers" is a valuable thing to mention and leave in thee imho
- "work to deliver value" indeed could go, doesn't add much
- "work from 'why'" is to a degree differentiating for us, but I suppose for brevity we could also leave it out

PS. Same comments hold for the Dutch version of course.

Reply:
As a general remark, I agree with most of your comments. However I think there's a lot of value in a more marketing-like style here: make a few, bold, memorable statements. The current style is quite long, technical, and nuanced. It makes me a bit hesitant to investing the time/effort of reading the whole thing; I only remember some key points anyway.

Maybe related is that it's not entirely clear what the goal of this document is. Maybe I envision it too much of something for potential clients? For that I would expect a very concise document with only a few bulletpoints.